qiskit/assembler/assemble_circuits.py

```diff
@@ -68,6 +68,7 @@ def _assemble_circuit(
 
     qreg_sizes = []
     creg_sizes = []
+    clbit_list = []
```

Comment:
Seems like this value is not used. Why this is defined here?

Reply:
Thanks for pointing this out. This was required in one of the previous commits but I see that this is now redundant. I'll remove this now.